tests/unit_tests/test_wave_06.py

```diff
@@ -32,13 +32,15 @@ def test_get_no_matching_items_by_category():
     )
 
     items = vendor.get_by_category("Electronics")
+   
+    assert items ==[]
```

Comment:
Spacing around comparison operators
```suggestion
    assert items == []
```